### ISSUE TITLE
Fix Compare Toolbar icons for higher resolution displays

### ISSUE THREAD

**Issue by ct1000** [jsleroy/compare-plugin#95]
![comparetoolbaricons](https://user-images.githubusercontent.com/32984005/52580796-f2709e80-2ddd-11e9-9ad8-69ba20b3ea9d.PNG)

Hi, this is a really useful plug-in and I use it frequently. If there's a way to fix the toolbar icons to support the higher resolution screens, that would be great. I've attached a picture of the Notepad++ toolbar and circled the compare icons. You'll see that all the other icons appear OK, but the compare icons are really small.

I don't know if this plug-in is being updated or not anymore. I hope it is. Thanks!

**Comment by i3fe-at-work** [comment]
Yes please do this! Also, keep up the great work on this useful plugin.

**Comment by Yaron10** [comment]
Implemented in Pavel Nedev's ComparePlus.
https://github.com/pnedev/compare-plugin.

**Comment by ct1000** [comment]
Hi,

 

I don’t understand “how” to get this update and install it. I’m new to looking at things on GitHub. Can you guide me in the right direction please? Thanks!

 

Best regards,

Christopher Throckmorton

**Comment by Yaron10** [comment]
Hello Christopher,

1. Make sure you're using the latest Notepad++ version.
2. Rename the `ComparePlugin` folder (Notepad++ menu -> Plugins -> Open Plugins Folder) to `ComparePlus` and delete the `ComparePlugin.dll`.
If you also have a sub-folder `ComparePlugin`, rename it too.
3. Go to https://ci.appveyor.com/project/pnedev/compare-plugin, select your platform (either Release x86 or Release x64), press `Artifacts` and download the `ComparePlus.dll`. 
4. Place `ComparePlus.dll` in the `ComparePlus` folder.

5. Go to https://github.com/pnedev/compare-plugin and press the `Star` button. :)

Let me know if it has worked for you.

Best regards.

**Comment by ct1000** [comment]
Hi,

 

I’m not sure what your name is, Yaron10 or Jsleroy. Anyway, hi and THANK YOU! I’m very grateful for your quick reply.

 

I followed your instructions and figured out how to install the ComparePlus plug-in as well as learned more about how to navigate the GitHub site. I really appreciate that!

 

I’m a C# .NET developer for some years now, but have NO experience with or how to use GitHub. However, I’m wanting very badly to move away from TFS and start using GitHub. I’d even like to be a contributor to some projects on there, but still need to figure out how to do that and how it all works.

 

Can you recommend a good place for me to start? I’m even wiling to pay someone for an hour or two of their time to help me. If you know someone that might be willing to do this, please let me know. Thanks!

 

PS I pressed the Star button and again thank you! 😊

 

Best regards,

Christopher Throckmorton

**Comment by Yaron10** [comment]
Hello Christopher,

You're quite welcome. It was a pleasure and I'm glad you've figured it out.
And thank you for pressing the `Star` button. Pavel Nedev (ComparePlugin / Plus developer) deserves many stars and thanks for his excellent work.

I'm not a developer. My main activity on GitHub is comments in Pavel Nedev's [ComparePlus repository](https://github.com/pnedev/compare-plugin).
You can press the `Watch` button as well and provide feedback via [Issues](https://github.com/pnedev/compare-plugin/issues).
I wish I could help you more.

Best regards.
Yaron
